### PR TITLE
Minor fixing

### PR DESCRIPTION
**Pull request by YAWAL**
Native Go-logger has been changed to custom logger from package loger.Log

### CODE REVIEW

**Review by justakit — APPROVED** [review]

File: messageService/message.go [hunk 1]
```diff
@@ -2,7 +2,7 @@ package messageService
 
 import (
 	"encoding/json"
-	"log"
+	"SSU-Golang-252-Chat/loger"
```
Comment (8tomat8) [inline messageService/message.go]: Here should be full package path.

File: messageService/message_test.go [hunk 2]
```diff
@@ -2,6 +2,7 @@ package messageService
 
 import (
 	"testing"
+	"SSU-Golang-252-Chat/loger"
```
Comment (8tomat8) [inline messageService/message_test.go]: Here should be full package path.

File: messageService/message_test.go [hunk 3]
```diff
@@ -21,46 +22,45 @@ func TestMarshalMessage(t *testing.T) {
 	header := MessageHeader{Type_, Command, UserName, Token}
 	body := MessageBody{ReceiverName, CurrentTime, Text}
 	message := Message{header, body}
-	mrsMsg, err := MarshalMessage(message)
+	mrsMsg, err := MarshalMessage(&message)
 	if err != nil {
-		t.Fatal("Error has occured: ", err)
+		loger.Log.Fatal("Error has occured: ", err)
```
Comment (8tomat8) [inline messageService/message_test.go]: Dont use app logger in tests. You have all required functionality in `t`

**Review by 8tomat8 — CHANGES_REQUESTED** [review]

File: messageService/message.go (hunk 1 above)
Comment (YAWAL) [reply messageService/message.go]: Done, full package path has been added.

File: messageService/message_test.go (hunk 2 above)
Comment (YAWAL) [reply messageService/message_test.go]: Import has been deleted: custom logger won't use in test file.

File: messageService/message_test.go (hunk 3 above)
Comment (YAWAL) [reply messageService/message_test.go]: Import has been deleted: custom logger won't use in test file.

**Review by 8tomat8 — APPROVED** [review]

**Pull request merged**